src/Signer/Hmac/Blake2b.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Signer\Hmac;
+
+use Lcobucci\JWT\Signer;
+use Lcobucci\JWT\Signer\Key;
+
+use function hash_equals;
+use function sodium_crypto_generichash;
+
+final class Blake2b implements Signer
+{
+    public function algorithmId(): string
+    {
+        return 'BLAKE2B';
```

Comment:
I was thinking that we should make the length visible and open for extension.

```suggestion
        return 'B2-256';
```

Reply:
I don't like this: the numbers seen in other algorithm IDs want to represent the algorithms used, not their output length; see for example `EdDSA`.

`Blake2b` has no variants, so there's only one name.

Reply:
That's fair, I'd argue that it only be Blake2 (as the 32 bytes output is compatible with Blake2s).

Reply:
I'd stick with `BLAKE2B`.

No matter what the output length is, BLAKE2B and BLAKE2S would produce different outputs.